app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt

```diff
@@ -99,7 +108,14 @@ class ProjectPageActivity :
         super.onCreate(savedInstanceState)
         binding = ActivityProjectPageBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        this.ksString = requireNotNull(environment().ksString())
+
```

Comment:
Add this line (only for activities): 
`        setUpConnectivityStatusCheck(lifecycle)
`
for the activity to register the Connectivity check, it gets lost when removing the inheritance from `BaseActivity`